test/validate-themes.js

```
@@ -0,0 +1,117 @@
+      }
+    }
+    
+    // Check for required UI colors
+    const missingColors = REQUIRED_UI_COLORS.filter(color => !theme.colors[color]);
+    if (missingColors.length > 0) {
+      console.warn(`  Warning: Missing UI colors: ${missingColors.join(', ')}`);
+    }
+    
+    // Validate color format (should be hex)
+    const colorRegex = /^#[0-9a-f]{6}([0-9a-f]{2})?$/i;
+    let invalidColors = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'invalidColors' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)